webpack/redux/reducers/RedHatRepositories/enabled.js

```diff
@@ -57,40 +58,46 @@ const changeRepoState = (state, repoToChange, stateDiff) => (
   )
 );
 
+// eslint-disable-next-line consistent-return
```

Comment:
Why disable this here?

Reply:
@waldenraines good catch, I don't think this is needed anymore.  I put it in before I added the return on line 76.